common/src/main/java/com/wynntils/core/features/overlays/OverlayManager.java

```diff
@@ -65,6 +58,14 @@ public static void onRenderPost(RenderEvent.Post event) {
     }
 
     private static void renderOverlays(RenderEvent event, OverlayInfo.RenderState renderState) {
+        boolean testMode = false;
+
+        if (McUtils.mc().screen instanceof OverlayManagementScreen screen) {
+            testMode = screen.isTestMode();
+        }
+
+        boolean shouldRender = !(McUtils.mc().screen instanceof OverlayManagementScreen);
```

Comment:
The reason you're not just bailing out here is that we might need to do setCanceled for all overlays?

Reply:
But you've already tested the instanceof above, so you can set shouldRender to true always and add a false assignment in the if body. That would make it more analogous to testMode.